app/services/moves/anonymiser.rb

```diff
@@ -2,26 +2,31 @@
 
 module Moves
   class Anonymiser
-    attr_accessor :nomis_offender_number, :day_offset, :move_response
+    attr_accessor :nomis_offender_number, :move
 
-    def initialize(nomis_offender_number:, day_offset:, move_response:)
-      self.nomis_offender_number = nomis_offender_number
-      self.day_offset = day_offset
-      self.move_response = move_response
+    def initialize(nomis_offender_number:, move:)
+      self.nomis_offender_number =
+        People::Anonymiser.encrypt_offender_number(offender_number: nomis_offender_number)
+      self.move = move
     end
 
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/AbcSize
     def call
-      start_date_time = move_response['startTime'].present? ? Time.parse(move_response['startTime']) : nil
+      from_location = Location.all.sample
+      to_location = Location.all.sample
+      start_date_time = move['startTime'].present? ? Time.parse(move['startTime']) : nil
       start_time = start_date_time&.strftime('%H:%M:%S') || '00:00:00'
 
-      move_response.merge(
-        offenderNo: nomis_offender_number,
-        judgeName: nil,
-        commentText: nil,
-        createDateTime: "<%= date.toISOString().split('.')[0] %>",
-        eventDate: "<%= date.toISOString().split('T')[0] %>",
-        startTime: "<%= date.toISOString().split('T')[0] + 'T' + '#{start_time}' %>"
+      move.merge(
+        person_nomis_prison_number: nomis_offender_number,
+        from_location_nomis_agency_id: from_location,
+        to_location_nomis_agency_id: to_location,
+        date: "<%= date.toISOString().split('T')[0] %>",
+        time_due: "<%= date.toISOString().split('T')[0] + 'T' + '#{start_time}' %>"
```

Comment:
I think we should still set the `judgeName` and `commentText` to `nil` even if we don't use them.

Reply:
AFAIK that wouldn't make a lot of sense, before we were anonymising the data that nomis was returning, I changed that to anonymise the data the client is returning for consistency on how for example the people anonymiser was working.
I wouldn't even be sure what parameter to add because that data is not what NomisClient#Moves is returning.